src/proof_system/permutation.rs

```diff
@@ -229,16 +276,105 @@ impl<F: PrimeField> ProverKey<F> {
                                // Sigma_4(X)
     }
 
+    /// Computes the lineariser check.
     fn compute_lineariser_check_is_one(
         &self,
         domain: &GeneralEvaluationDomain<F>,
         z_challenge: F,
         alpha_sq: F,
         z_coeffs: &DensePolynomial<F>,
     ) -> DensePolynomial<F> {
-        // Evaluate l_1(z)
         let l_1_z = domain.evaluate_all_lagrange_coefficients(z_challenge)[0];
-
         z_coeffs * (l_1_z * alpha_sq)
     }
 }
+
+/// Permutation Verifier Key
+#[derive(CanonicalDeserialize, CanonicalSerialize, derivative::Derivative)]
+#[derivative(
+    Clone(bound = ""),
+    Debug(bound = ""),
+    Eq(bound = ""),
+    PartialEq(bound = "")
+)]
+pub struct VerifierKey<E>
+where
+    E: PairingEngine,
+{
+    /// Left Permutation Commitment
+    pub left_sigma: Commitment<E>,
+
+    /// Right Permutation Commitment
+    pub right_sigma: Commitment<E>,
+
+    /// Output Permutation Commitment
+    pub out_sigma: Commitment<E>,
+
+    /// Fourth Permutation Commitment
+    pub fourth_sigma: Commitment<E>,
```

Comment:
Why just containing the Sigma commitments?

Reply:
I didn't change this, I just moved the prover and verifier keys into the same file, see the `master` branch: https://github.com/rust-zkp/ark-plonk/blob/33b8b7456edeab82e1ad39bd8678fff2a3a22f57/src/proof_system/widget/permutation/verifierkey.rs.